tests/conftest.py

```diff
@@ -7,7 +7,7 @@
 from tests.util import upload_image
 
 from tenable_io.client import TenableIOClient
-
+from tests.config import TenableIOTestConfig
 
 class App:
```

Comment:
`tests/conftest.py:12:1: E302 expected 2 blank lines, found 1` per pep8.

Reply:
Are you using a pylintrc file? This did not appear when I ran pylint in python3. I've updated all the files I touched to be two lines before the start of a class.

Reply:
We don't have a `pylintrc` file setup. We run `pep8 tenable_io tests` (you can see that in `.travis.yml`).